yarn-project/circuits.js/src/utils/call_wasm.ts

```diff
@@ -13,8 +13,8 @@ export async function callAsyncWasm<T>(
   // Allocate memory for the input buffer and the pointer to the pointer to the output buffer
   const inputBufPtr = wasm.call('bbmalloc', inputBuf.length);
   wasm.writeMemory(inputBufPtr, inputBuf);
-  if (method === 'merge_rollup__sim') {
-    // TODO: Remove this once base/root also return circuit failures
+  if (method.indexOf('_rollup_') != -1) {
+    // TODO: Remove this once kernel circuits also return failures
```

Comment:
If we might always want to keep the old way of making wasm calls, we might just want two functions? Not necessary to do that now, but just wanted to mention it.

Reply:
So actually this function is only used for kernel and rollups -> by my next PR, we can remove this :D